widget/ui/src/components/List/List.tsx

```diff
@@ -1,12 +1,13 @@
 import type { ListPropTypes } from './List.types';
+import type { PropsWithChildren } from 'react';
 
 import React from 'react';
 
 import { ListItem } from '../ListItem';
 
 import { BaseList } from './List.styles';
 
-function List(props: ListPropTypes) {
+function List(props: PropsWithChildren<ListPropTypes>) {
```

Comment:
I think by adding  'children' prop there is no need for 'items' prop anymore and we should use only one of these approaches.